artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/cursor/impl/PageSubscriptionImpl.java

```diff
@@ -1315,9 +1315,7 @@ private PagedReference moveNext() {
                   }
                }
 
-               if (!ignored) {
-                  position = message.getPosition();
-               }
+               position = message.getPosition();
```

Comment:
I think this made sense at some point, but after a few fixes later it's harmless.

I don't think this needs a JIRA as there's no issue I think.

I will merge and keep the JIRA, but I will rename the commit from being a question into something more affirmative.